tests/multinode.rs

```diff
@@ -1976,7 +1994,7 @@ fn test_two_fullnodes_rotate_every_second_tick() {
 
 #[test]
 fn test_one_fullnode_rotate_every_tick_with_transactions() {
-    test_fullnode_rotate(1, 1, false, true);
+    test_fullnode_rotate(2, 2, false, true);
```

Comment:
With `2, 2`, the test implementation is now out of sync with the test name `test_one_fullnode_rotate_every_tick_with_transaction`.  Please make them match again